src/main/java/org/springframework/data/couchbase/core/query/OptionsBuilder.java

```diff
@@ -370,10 +385,8 @@ static String toString(MutateInOptions o) {
 		return s.toString();
 	}
 
-	private static JsonObject getQueryOpts(QueryOptions.Built optsBuilt) {
-		JsonObject jo = JsonObject.create();
-		optsBuilt.injectParams(jo);
-		return jo;
+  public static JsonObject getQueryOpts(QueryOptions.Built optsBuilt) {
```

Comment:
It doesn't feel great turning QueryOptions into JSON, and it also calls into the internal method ClassicCoreQueryOps.convertOptions,   Luckily, I don't think this getQueryOpts is necessary anymore.  It's called from two places:

1. buildQueryOptions.  Which only uses it for getScanConsistency.  But that's not needed anymore.  QueryOptions.Built now implements CoreQueryOptions so you can call scanConsistency() directly on it.
2. buildTransactionQueryOptions.  As mentioned above I already feel a bit queasy about this solution of converting a QueryOptions into JSON and then using that to set raw parameters on a TransactionQueryOptions.  And now with CoreQueryOptions  you don't need it - you can copy over the parameters one by one (now including the serializer).